index.js

```diff
@@ -2,82 +2,91 @@
+                var message = 'Could not find ' + repositoryName + ' in ' + repositoryNames;
+                dateLog(message);
+                next(new Error(message));
+            }
 
-    repositoryNames.forEach(function (repositoryName) {
-        dateLog('Listening to ' + repositoryName);
-        webHookHandler.on(repositoryName, function (event, jsonResponse) {
             dateLog('Received event to repository: ' + repositoryName);
             dateLog('event: ' + event);
-            dateLog('jsonResponse: ' + jsonResponse);
+            dateLog('jsonResponse: ' + res.body);
 
-            var promise = Promise.resolve();
             var repositorySettings = Settings.repositories[repositoryName];
             var headers = {
                 'User-Agent': 'cesium-concierge',
                 Authorization: 'token ' + repositorySettings.gitHubToken
             };
             var checkChangesMd = repositorySettings.checkChangesMd;
-
-            if ((event === 'issues' || event === 'pull_request') && jsonResponse.action === 'closed' &&
+            var promise = Promise.resolve();
+            if ((event === 'issues' || event === 'pull_request') && res.body.action === 'closed' &&
                 repositorySettings.remindForum) {
                 promise = promise.then(function () {
                     dateLog('Calling commentOnClosedIssue');
-                    return commentOnClosedIssue(jsonResponse, headers);
+                    return commentOnClosedIssue(res.body, headers);
                 });
-            } else if (event === 'pull_request' && jsonResponse.action === 'opened' &&
+            } else if (event === 'pull_request' && res.body.action === 'opened' &&
                 (defined(repositorySettings.thirdPartyFolders) || checkChangesMd)) {
                 promise = promise.then(function () {
                     dateLog('Calling commentOnOpenedPullRequest');
-                    return commentOnOpenedPullRequest(jsonResponse, headers, repositorySettings.thirdPartyFolders,
+                    return commentOnOpenedPullRequest(res.body, headers, repositorySettings.thirdPartyFolders,
                         checkChangesMd);
                 });
             }
 
-            promise.then(function (res) {
-                if (!defined(res)) {
+            promise.then(function (result) {
+                res.status(200).send({
+                    success: true
+                });
+
+                if (!defined(result)) {
                     dateLog('GitHub request did not match any events the server is listening for');
                     return;
                 }
-                dateLog('GitHub API returned with statusCode: ' + res.statusCode);
-                dateLog('and statusMessage: ' + res.statusMessage);
+                dateLog('GitHub API returned with statusCode: ' + result.statusCode);
+                dateLog('and statusMessage: ' + result.statusMessage);
```

Comment:
In most cases, yes, but there are edge cases where not calling `next` can be an issue, so it's just a good practice to always call next.